lib/SPIRV/SPIRVReader.h

```diff
@@ -159,6 +159,7 @@ class SPIRVToLLVM {
   SPIRVToLLVMPlaceholderMap PlaceholderMap;
   std::unique_ptr<SPIRVToLLVMDbgTran> DbgTran;
   std::vector<Constant *> GlobalAnnotations;
```

Comment:
It would be nice to have a comment about GlobalAnnotations and AnnotationsMap fields to understand what is the difference between them.

Reply:
Done. Looks better now?

Reply:
Ok, let me rephrase this: May we join GlobalAnnotations and AnnotationsMap? And why if we cann't?

Reply:
Do you mean to use only one container instead of these two?

Reply:
Yes.

Reply:
Done